src/app/component/CategoryPagination/CategoryPagination.style.scss

```diff
@@ -26,22 +25,22 @@
             content: '';
         }
 
-        @include mobile {
-            &:first-child {
+        &:first-child {
+            @include mobile {
                 padding-left: 0;
             }
         }
 
         &:last-child {
             margin-bottom: 0;
         }
+    }
+}
 
-        a {
-            color: var(--color-pagination-link-text);
+.PaginationLink {
```

Comment:
Please make sure you do not have more then one parent selector declared in .style. file! Can you make this a child element (in terms of BEM) of `CategoryPagination` block?